storage/map_files_downloader.cpp

```diff
@@ -161,15 +163,20 @@ MapFilesDownloader::ServersList MapFilesDownloader::LoadServersList()
     request.RunHttpRequest(httpResult);
   }
 
-  std::vector<std::string> urls;
-  downloader::GetServersList(httpResult, urls);
-  CHECK(!urls.empty(), ());
-  return urls;
+  std::optional<MetaConfig> metaConfig;
+  if (httpResult.empty() || !(metaConfig = downloader::ParseMetaConfig(httpResult)))
+  {
+    metaConfig = downloader::ParseMetaConfig(GetPlatform().DefaultUrlsJSON());
+    VERIFY(metaConfig, ());
+    LOG(LWARNING, ("Can't get meta configuration from request, using default servers:", metaConfig->m_serversList));
```

Comment:
LERROR ?

Reply:
Maybe not a bad idea. Initially it was WARNING, but I can increase to ERROR.

Reply:
I am not sure here. This logic uses `DEFAULT_URLS_JSON` if metaserver is not available. This is a legitimate situation.

Reply:
Ok, keep LWARNING.